tardis/model/base.py

```diff
@@ -0,0 +1,14 @@
+class Radial1DModel(object):
```

Comment:
Regarding the density, apparently descriptors are not the right option here, a descriptor must be a class attribute, and not an instance attribute.

So it would look something like this:
```python
class Model(object):
    density = HomologousDensity(density_0=xyz, time_0=xyz)
    def __init__(self, ...):
        ....
```

which is not what we want, since we need `density_0` and `time_0` to be different for each model instance. If I put the density assignment line inside the init method, the descriptor will not work.